freegeek-info.sh

```diff
@@ -154,27 +160,33 @@ echo ""
 # Port stuff
 
 # SD Card
-mmc=$(sudo dmesg | grep -i mmc)
+mmc=$(dmesg | grep -i mmc)
 if [[ -n "$mmc" ]]; then
-    echo -e "${BOLD}SD Card slot:${RESET} Probably"
+    echo -e "${BOLD}SD Card Slot:${RESET} Probably"
 else
-    echo -e "${BOLD}SD Card slot:${RESET} No"
+    echo -e "${BOLD}SD Card Slot:${RESET} No"
 fi
 
 # USB 3.0
-usb3=$(lsblk | grep 3.0)
+usb3=$(lsusb | grep 3.0)
 if [[ -n "$usb3" ]]; then
-    echo -e "${BOLD}USB3.0:${RESET} Probably"
+    echo -e "${BOLD}USB3.0:${RESET} Probably (Check for blue USB ports manually)"
```

Comment:
```suggestion
    echo -e "${BOLD}USB3.0:${RESET} Probably (check!)"
```
You can have 3.0 without a blue accented port, and it could also pull the usbc as 3.0

Reply:
True.